addons/base_import/static/tests/import_action.test.js

```diff
@@ -278,11 +281,17 @@ describe("Import view", () => {
             }
         );
         expect(".o_import_action").toHaveCount(1);
-        expect(".o_nocontent_help .btn-outline-primary").toHaveText("Some Import Template");
-        expect(".o_nocontent_help .btn-outline-primary").toHaveProperty(
+        expect(".o_nocontent_help .btn-outline-primary").toHaveCount(2);
+        expect(".o_nocontent_help > div:nth-of-type(2) .btn-outline-primary").toHaveText("Some Import Template");
+        expect(".o_nocontent_help > div:nth-of-type(2) .btn-outline-primary").toHaveProperty(
             "href",
             "https://www.hoot.test" + templateURL
         );
+        expect(".o_nocontent_help > div:nth-of-type(3) .btn-outline-primary").toHaveText("Another Import Template");
+        expect(".o_nocontent_help > div:nth-of-type(3) .btn-outline-primary").toHaveProperty(
```

Comment:
@Iucapad i'm not a fan of these selectors, as they rely on the number of divs in `.o_nocontent_help`, but as far as I can tell i need the selector for `expect` to find exactly one element (e.g. i cannot do `expect('selector')[0].toHaveText('sthg')`)

if there's a cleaner way, i'm all 👂 !